Tests seem to pass on CI, but fail when using make locally

### Description


### Steps to Reproduce
1. git checkout master
2. make test
3. Tests fail

**Expected behavior:**
- Tests should fail on CI if they fail via the make file

**Actual behavior:**
- Tests pass on CI https://github.com/arr-ai/arrai/pull/102/checks?check_run_id=569350764


* Debug&Crash Output
Test failure output
```
go test  -tags timingsensitive ./...
ok  	github.com/arr-ai/arrai/cmd/arrai	0.380s
?   	github.com/arr-ai/arrai/engine	[no test files]
ok  	github.com/arr-ai/arrai/rel	0.042s
--- FAIL: TestImportCache (0.14s)
    import_cache_test.go:51:
        	Error Trace:	import_cache_test.go:51
        	Error:      	Not equal:
        	            	expected: []string{"10 > a 1", "30 > b", "40 > c", "50 > d 1", "80 < b", "90 < d 1", "90 > d 2", "110 < a 1", "130 < d 2", "140 < c"}
        	            	actual  : []string{"10 > a 1", "30 > b", "40 > c", "50 > d 1", "80 < b", "90 < d 1", "90 > d 2", "110 < a 1", "132 < d 2", "142 < c"}

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -9,4 +9,4 @@
        	            	  (string) (len=9) "110 < a 1",
        	            	- (string) (len=9) "130 < d 2",
        	            	- (string) (len=7) "140 < c"
        	            	+ (string) (len=9) "132 < d 2",
        	            	+ (string) (len=7) "142 < c"
        	            	 }
        	Test:       	TestImportCache
FAIL
FAIL	github.com/arr-ai/arrai/syntax	10.288s
?   	github.com/arr-ai/arrai/tools/module	[no test files]
ok  	github.com/arr-ai/arrai/translate	0.092s
FAIL
make: *** [test] Error 1
```

### Your Environments
```sh
$ go version
go version go1.13.9 darwin/amd64

```

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,5 @@
 all: test lint wasm
 
-# TODO: If this Makefile is ever used for CI, suppress timingsensitive there.
 test:
 	go test $(GOTESTFLAGS) -tags timingsensitive ./...
 

--- a/syntax/import_cache_test.go
+++ b/syntax/import_cache_test.go
@@ -19,7 +19,7 @@ func TestImportCache(t *testing.T) {
 	log := func(format string, args ...interface{}) {
 		m.Lock()
 		defer m.Unlock()
-		msgs = append(msgs, fmt.Sprintf("%d ", int(time.Since(start).Milliseconds()))+fmt.Sprintf(format, args...))
+		msgs = append(msgs, fmt.Sprintf(format, args...))
 	}
 
 	cache := newCache()
@@ -50,16 +50,16 @@ func TestImportCache(t *testing.T) {
 
 	assert.Equal(t,
 		[]string{
-			"10 > a 1",
-			"30 > b",
-			"40 > c",
-			"50 > d 1",
-			"80 < b",
-			"90 < d 1",
-			"90 > d 2",
-			"110 < a 1",
-			"130 < d 2",
-			"140 < c",
+			"> a 1",
+			"> b",
+			"> c",
+			"> d 1",
+			"< b",
+			"< d 1",
+			"> d 2",
+			"< a 1",
+			"< d 2",
+			"< c",
 		},
 		msgs,
 	)

